acestep/audio_utils.py

```
@@ -181,18 +197,31 @@ def _save_mp3(
+            except Exception:
+                logger.error(f"[AudioSaver] {reason} Additionally failed to preserve WAV fallback.")
+                raise RuntimeError(reason) from e
+
+            logger.warning(f"[AudioSaver] {reason} Saved WAV fallback to {wav_fallback_path} instead.")
+            raise AudioExportDegradedError(reason, str(wav_fallback_path), "mp3") from e
         finally:
             try:
                 temp_wav_path.unlink(missing_ok=True)
             except Exception:
-                logger.warning(f"[AudioSaver] Failed to remove temporary WAV file: {temp_wav_path}")
+                pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)